### DISCUSSION TITLE
Classic Explorer Issue

### DISCUSSION THREAD

**Discussion by phil-co** [Open-Shell/Open-Shell-Menu#678] [Q&A] [answered]
It appears that the custom button toolbar provided by Classic Explorer has stopped working in the latest update to Window 10. Other features like the status bar still work, so I guess it's something Microsoft changed in Windows Explorer that is blocking the toolbar.

Hoping it can be fixed soon, because I feel lost without it...

Thanks!

**Comment by davidpierson** [comment]
Hmmm, Classic Explorer Bar is working for me. In Windows Explorer or File Explorer as I notice it calls itself.
My event sequence: Windows Home upgraded itself to 20H2 build 19042.  Classic Shell stopped working. Uninstalled that and added Open Shell 4.4.160. Explorer Bar is showing and hiding as expected.
I wish it had a Map Network Drive button, but that aside, it works okay here. 
It is an option during installation; why not try uninstall and re-install?
Edit: playing around with options, I discover I can add buttons to it. So thank you, I learned something new today.

**Reply by phil-co** [reply]
Sorry for the very late reply. I tried both the solutions below, which has solved the problem, though I'm not exactly sure which was the precise solution, but I think it was enabling the Classic Explorer bar from view options, which I don't recall ever having to do before (but I could be totally wrong about that). Anyway, thanks for the help!

**Comment by blackcrack** [comment]
164 is actual
plz test this : https://ci.appveyor.com/project/passionate-coder/open-shell-menu/build/artifacts
best

**Reply by phil-co** [reply]
This works, although I'm not 100% sure if it wasn't the suggestion below by @bonzibudd that did the trick.

**Comment by bonzibudd** [comment, marked as answer]
In Explorer, try going to View -> Options and click the dropdown arrow. There should be an option called "Classic Explorer Bar".

**Reply by phil-co** [reply]
I think this suggestion is what actually got it working for me, because I don't remember ever going into the view options to enable the Classic Explorer bar. Thanks!
